src/ui/settings/ViewGiveFeedback.qml

```diff
@@ -375,7 +378,7 @@ Item {
                        anchors.top = undefined;
                        anchors.topMargin = undefined;
                        anchors.bottom= parent.bottom
-                       anchors.bottomMargin = VPNTheme.theme.windowMargin * 4
+                       anchors.bottomMargin = VPNTheme.theme.navBarHeightWithMargins
```

Comment:
For non-scrolling page views, we will have to remember to account for the navbar by anchoring the bottom margin of the layout to this value. Alternatively, we can create a new base component (maybe something like `SinglePageView`) that has this built in so we don't have to remember to add it for every new page view, and can be the root component of all of our pages. That's a decent amount of churn so I haven't done that yet, but would like to get some thoughts

Reply:
I like the idea and I’m also wondering if we rather should have all views in a `PageView` that can be potentially scrollable. If we feel like that is too much work for now that would be probably a good task to add to a design system epic.